### ISSUE TITLE
this code runs very slow!

### ISSUE THREAD

**Issue by viekie** [rtqichen/style-swap#36]
@rtqichen @Naruto-Sasuke @LambdaWill 
i run the code in cpu mode. I find that 

img_latent = criterion.net.output:clone()   and  swap_latent = swap:forward(img_latent):clone() are very slow, it takes several mimutes, why?

any one can help me  ?

**Comment by rtqichen** [comment]
Yeah, the swap:forward on cpu is very slow, but with cuda it should be okay. Do you have a Nvidia GPU?

**Comment by viekie** [comment]
I have a TitanX GPU in my pc, but the mechine which I want to run the code is  a virtual mechine on aliyun cloud platform(just like amazon EC2), do you have some suggestions to  speed up ? @rtqichen @Naruto-Sasuke @LambdaWill
